.github/workflows/gotest.yml

```diff
@@ -14,5 +17,13 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: '1.22.11'
-      # TODO: add coverage
-      - run: go test -timeout=5m -race -tags=verify_logs ./...
+      - run: go test -coverprofile=coverage.txt -timeout=5m -race -tags=verify_logs ./...
```

Comment:
is 5m enough for the entire running?

Reply:
For now, it looks good. When we have more or longer tests, we can increase it then.